packages/longhorn-1.2/templates/crd-template/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+version: 1.2.0
```

Comment:
Just an fyi that only this version will be airgapped, not 1.1.x.

Reply:
I thought all feature charts in cluster explorer (2.6 or 2.5) all air-gapped. 

Could u explain more? What case will be air-gapped and what case will not?

Reply:
Only the latest version of any chart is air-gapped. There is an issue to add support where it goes off of the rancher-version instead, so you could have multiple if they both specify this upcoming version.

Reply:
Got it. But how do we handle multiple releases case not just one latest version? Like I mentioned above if we have ongoing releases for more than one release line like 1.1.x and 1.2.x?

Reply:
We can't, we only airgap one line for now. So you can release both of them in 2.6.1, but only 1.2 will get air-gapped.

Reply:
Thanks for the explanation. One more question, do we have any documentation about this? I believe some people will have similar questions in the future.

Reply:
Not that I'm aware of. There is an issue to allow multiple per release so it will change soon anyway.

Reply:
Ok this part is clear now, resolved